tests/factories/module_item_configuration.py

```diff
@@ -0,0 +1,13 @@
+from factory import Faker, make_factory
+from factory.alchemy import SQLAlchemyModelFactory
+
+from lms import models
+from tests.factories.attributes import RESOURCE_LINK_ID, TOOL_CONSUMER_INSTANCE_GUID
+
+ModuleItemConfiguration = make_factory(
+    models.ModuleItemConfiguration,
+    FACTORY_CLASS=SQLAlchemyModelFactory,
+    resource_link_id=RESOURCE_LINK_ID,
+    tool_consumer_instance_guid=TOOL_CONSUMER_INSTANCE_GUID,
+    document_url=Faker("uri"),
```

Comment:
The only idea I can come up with is, would we want to make this randomly spit out our "blackboard:" style URIs?